### ISSUE TITLE
configure size of laser pointer

### ISSUE THREAD

**Issue by pank** [Cimbali/pympress#250]
**Is your feature request related to a problem? Please describe.**
The laser pointer can be hard to see when giving presentations.

**Describe the solution you'd like**
It would be nice if the size of the laser pointer could be configured.
It would be fine to just have a single size to configure in the config file. Alternatively, the size could be changed dynamically in the presentation. 

**Describe alternatives you've considered**
I guess via the highlight mode, but it's a bit heavy for just pointing and, at least on windows system, it is a bit slow.

**Comment by Cimbali** [comment]
What’s your resolution and is there any scaling? I think the pointers are fixed size (in pixels) currently so they may appear smaller on larger resolutions. Maybe scaling the pointer to the slide/window size is enough rather than (or additionally to) making it configurable?

**Comment by pank** [comment]
but the pointer looks like it's only a couple of pixels my screen. So maybe there's something funky going on.

**Comment by Cimbali** [comment]
Here’s what it looks like on my 1920x1080, is yours smaller?
![image](https://user-images.githubusercontent.com/6126377/188921304-f9df99fc-2391-486f-8cb4-ff74db7abde1.png)

**Comment by pank** [comment]
yes, it's just a dot. could it be issue #248 that is causing this...
